src/components/animations/ButtonPressAnimation/ButtonPressAnimation.android.js

```diff
@@ -297,9 +292,9 @@ export default function ButtonPressAnimation({
       transformOrigin={normalizedTransformOrigin}
       wrapperStyle={wrapperStyle}
     >
-      <Content pointerEvents="box-only" style={style}>
+      <View pointerEvents="box-only" style={[{ overflow: 'visible' }, style]}>
```

Comment:
maybe we can move `{ overflow: 'visible' }` to `sx`

Reply:
Done!